Dockerfile

```diff
@@ -0,0 +1,55 @@
+# https://docs.docker.com/samples/rails/
+FROM ruby:2.7.3
+ 
+WORKDIR  /jinglejam
+
+COPY Gemfile /jinglejam/Gemfile
+COPY Gemfile.lock /jinglejam/Gemfile.lock
+RUN gem install bundler:2.2.16
+RUN bundler install
+RUN bundle install
+
+# Replace shell with bash so we can source files
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# Set debconf to run non-interactively
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Install base dependencies
+RUN apt-get update && apt-get install -y -q --no-install-recommends \
+        apt-transport-https \
+        build-essential \
+        ca-certificates \
+        curl \
+        git \
+        libssl-dev \
+        wget \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 14.16.1
+
+
+RUN mkdir -p $NVM_DIR
+
+# install nvm
+# https://github.com/creationix/nvm#install-script
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
```

Comment:
If you're going to install nvm using a helper script retrieved via curl/wget, it should checksum validated (to help prevent external tampering)

Reply:
Any suggestions on how to do that?